interfaces/cython/cantera/_cantera.pxd

```diff
@@ -126,6 +126,7 @@ cdef extern from "cantera/thermo/ThermoPhase.h" namespace "Cantera":
 
         # miscellaneous
         string type()
+        string getPhase()
```

Comment:
Needs `except +translate_exception` to avoid crashes when handling errors.

Reply:
Thanks, I should probably add a test case for the error.